.github/actions/e2e/wireguard.yaml

```diff
@@ -74,3 +74,17 @@
   skip-upgrade: 'true'
   # L7 testing: WireGuard encryption + Ingress controller requires L7 proxy for HTTP routing
   test-l7: 'true'
+- name: 'wireguard-6'
```

Comment:
Couldn't we merge this config with `wireguard-1`?

Reply:
No, because we need to skip the upgrade on this test config for now. When upgrading we will see failure for obvious reasons. Still need to add a comment to the upgrade guide for that though.